timelock-agent/src/main/java/com/palantir/timelock/paxos/AsyncTimeLockServicesCreator.java

```diff
@@ -102,6 +102,7 @@ private static AsyncTimelockService createRawAsyncTimelockService(
     }
 
     private static <T> T instrument(Class<T> serviceClass, T service, String client) {
-        return AtlasDbMetrics.instrument(serviceClass, service, MetricRegistry.name(serviceClass, client));
+        // TODO(nziebart): tag with the client name, when tritium supports it
```

Comment:
This exists now as of tritium-tags 0.7.0-beta4, see https://github.com/palantir/tritium/releases

We'll need to do a fairly major cleanup of Atlas metric names which I'm hoping to get a PR for sometime this week, and have a proper 0.7.0 release to depend upon.